asv_bench/benchmarks/timeseries.py

```diff
@@ -381,6 +386,36 @@ def time_format_exact(self):
     def time_format_no_exact(self):
         to_datetime(self.s, format='%d%b%y', exact=False)
 
+    def time_cache_true_with_unique_seconds_and_unit(self):
```

Comment:
ok, I think there is a actually a way to do this with an asv matrix, but can ok for now